### ISSUE TITLE
[Bug]: Docking frames outside of main windows looks funky

### ISSUE THREAD

**Issue by Tdue21** [RPTools/maptool#3408]
### Describe the Bug

With a multiple monitor setup, it is possible to place frames etc. outside of the main window for maximizing map viewport. 
The result is less than aesthetical pleasing to look at. 

I realize it is probably an artifact of the docking system, but hopefully something that can be tweaked by someone than actually understands Java (i.e. not me ;) ).

When two or more frames are docked together outside of the main windows, they are first framed by a Windows XP looking inner window, and then the native window (Windows 10, in my case). 

### To Reproduce

1. Drag two frames outside of the main window
2. Dock the two frames together.
3. Observe a funky looking window.

### Expected Behaviour

Basically what happens now, but without the extra Windows XP looking inner window. 

### Screenshots

![image](https://user-images.githubusercontent.com/2953375/163192244-a34bddb9-9d48-4a64-ba5d-ac35a7a25ce2.png)

### MapTool Info

1.11.5

### Desktop

Windows 10

**Comment by kwvanderlinde** [comment]
This is the same as #2455 which was fixed in 1.14.
